src/main/java/make_comment/driver/MakeCommentDatabaseAccess.java

```diff
@@ -0,0 +1,136 @@
+
+
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public void setNumComments(int newNumCommentCreated) {
+
+        File file = fileGetter("numCommentsCreated.csv");
+
+        try {
+            FileReader fileReader = new FileReader(file);
+
+            CSVReader csvReader = new CSVReader(fileReader);
+
+            List<String[]> csvBody = csvReader.readAll();
+            csvBody.get(1)[0] = String.valueOf(newNumCommentCreated);
+            FileWriter outputFile = new FileWriter(file);
+            CSVWriter writer = new CSVWriter(outputFile);
+            writer.writeAll(csvBody);
+            writer.flush();
+            writer.close();
+            csvReader.close();
+        } catch (IOException | CsvException e) {
+            System.out.println("file not found or incorrect format, comment is not saved");
+        }
+    }
+
+    @Override
+    public void saveComment(Map<String, String> commentAttributes) {
+        File file = fileGetter("comments.csv");
+        String[] commentAtt = new String[commentAttributes.size()];
+        commentAtt[0] = commentAttributes.get("commentID");
+        commentAtt[1] = commentAttributes.get("commenterID");
+        commentAtt[2] = commentAttributes.get("body");
+        commentAtt[3] = commentAttributes.get("creationDate");
+
+        try {
+            FileReader filereader = new FileReader(file);
+            CSVReader csvReader = new CSVReader(filereader);
+
+            List<String[]> csvBody = csvReader.readAll();
+            csvBody.add(commentAtt);
+            FileWriter outputFile = new FileWriter(file);
+            CSVWriter writer = new CSVWriter(outputFile);
+            writer.writeAll(csvBody);
+            writer.flush();
+            writer.close();
+            csvReader.close();
+        } catch (IOException | CsvException e) {
+          System.out.println("file not found or incorrect format, comment is not saved");
+        }
+
+    }
+
+    @Override
+    public void updatePostDB(List<String[]> updatedPosts) {
+        File file = fileGetter("posts.csv");
+        try {
+            FileWriter filewriter = new FileWriter(file);
+            CSVWriter writer = new CSVWriter(filewriter);
+            writer.writeAll(updatedPosts);
+            writer.flush();
+            writer.close();
+        } catch (IOException e) {
+            System.out.println("file not found or incorrect format, comment is not saved");
+        }
+
+
+    }
+
+
+    @Override
+    public List<String[]> getCurrentPosts() {
+
+        File file = fileGetter("posts.csv");
+
+        try {
+            FileReader fileReader = new FileReader(file);
+            CSVReaderBuilder csvReaderBuilder = new CSVReaderBuilder(fileReader);
+            CSVReader reader = csvReaderBuilder.build();
+            List<String[]> postData = reader.readAll();
+            // Remove the row corresponding to the header
```

Comment:
Was the header ever removed?

Reply:
I suppose it isnt necessary as this gets handled elsewhere, so not actually an issue. May want to remove the comment tho